performance/bulk-edit.php

```diff
@@ -25,9 +25,22 @@ function defer_term_counting() {
 function bulk_editing_is_limited() {
 	$per_page = get_query_var( 'posts_per_page' );
 
+	// Get total number of entries
+	$total_posts = $GLOBALS['wp_query']->found_posts;
```

Comment:
I wonder if we should verify that `$GLOBALS['wp_query']` is actually a `WP_Query` object in case someone accidentally calls this function where they're not supposed to?